### ISSUE TITLE
Ratio for different resolutions

### ISSUE THREAD

**Issue by YairShachar** [YairShachar/Intray#7]
Fix issue of the tray not adjusting to different resolutions, i.e
![img7039042850952271655](https://user-images.githubusercontent.com/25391023/133938936-276074ec-f802-4121-b8e9-af86fa306abd.png)

This is because dimensions are hardcoded, need to make them resolution dependant

**Comment by knoxy5467** [comment]
Hi I was looking to try to solve this issue as my first issue on github, which file is it located in?

**Comment by YairShachar** [comment]
Hi, very happy to hear that you are interested in solving an issue here,
On second thought it might not be so simple as to be a first ever issue to solve,
but if you feel that you are able to do it it would be wonderful :)

From what I understand, a good starting point would be 
https://stackoverflow.com/questions/22838357/is-it-possible-to-make-swing-based-application-independent-of-screen-resolution

This means that (atleast) the classes: 
IntraySubmitEntryPanel
IntraySubmitEntryButton
IntrayTextField
EntryListFrame
IntrayPrefsDialog

should be able to adjust their sizes according to the resolution.

We would also need to deal with the right click menu items which are located in 
IntraySettingsMenu,
setting the item fonts & sizes according to the resolution.

(i.e isLockedCheckBox.setFont(...))



If you are willing and starting to work on this issue please feel free to concat and ask questions if you deal with any problems :)
If you think you have solved it please publish a PR

**Comment by knoxy5467** [comment]
Yup! I was just wondering what the ratio was supposed to be aswell as I'm
getting an error when I try to build it. I'll post the error in a bit if
you don't mind. It's probably something simple that I missed.

**Comment by YairShachar** [comment]
Sure feel free. to me on a 27'' at 1920 x 1080 the ratio of 125 on 25 for the IntrayTextField seems nice.
